rules/react/walmart.js

```diff
@@ -68,7 +68,7 @@ module.exports = {
     // Prevent usage of unknown DOM property
     "react/no-unknown-property": 2,
     // Enforce ES5 or ES6 class for React Components
-    "react/prefer-es6-class": 0,
+    "react/prefer-es6-class": 2,
     // Prevent missing props validation in a React component definition
     "react/prop-types": 2,
```

Comment:
Please make a separate issue - discussion here should be limited to the rules that were added in this PR